utils/enums.py

```diff
@@ -10,3 +10,9 @@ class ActivationType(enum.IntEnum):
     MINUTE = 6
     HOUR = 7
     SPECIAL = 8
+
+
+class CoinsAutoConvert(enum.IntEnum):
+    ASK = 0
+    ALWAYS = 1
+    NEVER = -1
```

Comment:
Why -1 instead of 2?

Reply:
It felt right at the time, because it was the opposite of ALWAYS, and I likened it to that of adv/dis